plugins/package-managers/node/src/main/kotlin/npm/Npm.kt

```diff
@@ -95,7 +95,7 @@ class Npm(override val descriptor: PluginDescriptor = NpmFactory.descriptor, pri
 
     private val handler = NpmDependencyHandler(moduleInfoResolver)
 
-    override val graphBuilder by lazy { DependencyGraphBuilder(handler) }
+    override val graphBuilder = DependencyGraphBuilder(handler)
```

Comment:
Commit message typo: s/chap/cheap/

Reply:
Sorry, I overlooked more typos in the title:
- s/unecessary/unnecessary/
- "execution" should probably say "evaluation"

Reply:
Missed, that. It's now fixed.